docs/pages/reference/cli.mdx

```diff
@@ -2373,6 +2396,113 @@ $ tctl alerts create \
     "The system is under maintenance, functionality may be limited."
 ```
 
+### tctl devices add
+
+Register a device.
+
+
+```code
+$ tctl devices add --os=OS --asset-tag=SERIAL_NUMBER
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--os` | none | `linux`, `macos`, `windows` | Device operating system |
```

Comment:
If we're listing all eventual values for `--os`, should we add a note to the reference that only `macos` is currently available?

Reply:
I skipped that on purpose here so there is less risk of forgetting to update it later. The guide does spell it out, as does the command if you try to use it today.